### ISSUE TITLE
segfault on Julia v1.5.3

### ISSUE THREAD

**Issue by felipenoris** [JuliaLang/PackageCompiler.jl#466]
This `precompile_execution_file` script segfaults when upgrading to Julia v1.5.3:

```julia
import FinancialDSL
include(joinpath(pkgdir(FinancialDSL), "test", "runtests.jl"))
```

FinancialDSL is a registered package. Sorry I'm unable to provide more detail right now. I'll update this later.

**Comment by Sacha0** [comment]
I imagine you are hitting a segfault related to `jl_static_show_{x,next}_`, and if so we have a fix inbound :).

**Comment by felipenoris** [comment]
@Sacha0 yes, exactly! Thanks!

**Comment by NHDaly** [comment]
Here is the fix, which will fix this for julia 1.6: https://github.com/JuliaLang/julia/pull/38399

Unfortunately, of course, this won't fix the breakage on v1.5.3, unless we do _yet another_ release as v1.5.4...

@KristofferC - if it's helpful, and people think it's worth it, @Sacha0 and I would be happy to try to take the lead on running one more point release to make it easier for you all? I'm super sorry that this broke things 😞. I guess you were right about the potential scariness in backporting a relatively recent PR to master 😬.

**Comment by KristofferC** [comment]
Probably need a 1.5.4 for https://github.com/JuliaLang/julia/issues/38411 anyway.

**Comment by KristofferC** [comment]
Should be fixed in 1.5.4 / 1.6.
